amida-tech/indaba-client
# Styling/inba 558 wrap question

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
changes the survey question input to a a text area so that text wraps. Adds focus and active state to question
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-558
#### How should this be manually tested?
go to a project that has a question on the suvey. Edit as html and fill the question with text that would exceed the length of the textarea. Ensure the text wraps and area scrolls if the height of input exceeds text area height
#### Background/Context

#### Screenshots (if appropriate):
